.github/workflows/coverity.yml

```diff
@@ -0,0 +1,28 @@
+name: Coverity scan
+on:
+  # run once daily at 00:30 UTC due to
+  # https://scan.coverity.com/faq#frequency
+  schedule:
+    - cron: "30 0 * * *"
+
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        id: dependencies
+        uses: ./.github/actions/install-dependencies
+
+      - name: Configure
+        id: configure
+        uses: ./.github/actions/configure
+
+      - name: Execute and submit coverity scan
+        uses: vapier/coverity-scan-action@v1.2.0
+        with:
+          email: "sssd-team@redhat.com"
+          token: ${{ secrets.COVERITY_SCAN_TOKEN }}
+          working-directory: x86_64
```

Comment:
Why is this needed?

Reply:
Using the `https://github.com/vapier/coverity-scan-action` composite action fails after 'reconfig' step when it attempts to run 'make' from the github working space directory. Github `working-directory` keyword is not supported with `uses` lines, therefore support was added to provide an input variable `working-directory` which sets this for run steps in the composite action itself.

This could all be done with `run: |` shell commands but I thought it is better/cleaner to extend and re-use the marketplace coverity-scan-action

Reply:
Ah, I didn't realize that `reconfig` creates x86_64 directory. It may be worth to modify the action to configure sssd in the github workspace because this may be an issue in the future again. Different directory may be optional.

Reply:
I  agree, I added this to my to-do list.